Octokit/Models/Response/Enterprise/PreReceiveEnvironment.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+using Octokit.Internal;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Describes a pre-receive environment.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class PreReceiveEnvironment
+    {
+        /// <summary>
+        /// Identifier for the pre-receive environment.
+        /// </summary>
+        public int Id { get; protected set; }
```

Comment:
Can we use `long` for the Id field, just in case we ever reach max int?  Probably wouldnt happen on pre receive environments, but as a general policy all Id fields should be `long` (we got bitten by PullRequest Status Ids in the past!)

Reply:
Good call, i'll update that